ci/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+start_fold_marker() {
+  echo "travis_fold:start:install.$1"
+}
+
+end_fold_marker() {
+  echo "travis_fold:end:install.$1"
+}
+
+start_fold_marker 'dependencies'
+sudo add-apt-repository ppa:jonathonf/binutils -y &&\
+  sudo add-apt-repository ppa:ondrej/autotools -y &&\
+  sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```

Comment:
Again, my lack of familiarity with Ubuntu, but I am not really comfortable with using autotools that are not in the base distro.